.jenkins/caffe2/test.sh

```
@@ -78,6 +78,7 @@ for test in $INSTALL_PREFIX/test/*; do
   tmp_exit_code="$?"
   if [ "$exit_code" -eq 0 ]; then
     exit_code="$tmp_exit_code"
+    exit "$exit_code"
```

in order to fail fast, we just need to remove line 62 and remove all these tmp_exit_code

---

Line 62 is an empty line..

---

sorry I probably starred at the wrong line, I was referring to the line `set +e`